ui/src/app/settings/components/repos-list/repos-list.tsx

```diff
@@ -161,7 +161,7 @@ export class ReposList extends React.Component<RouteComponentProps<any>> {
                                     <FormField formApi={formApi} label='Skip server verification' field='insecure' component={CheckboxField}/>
                                 </div>
                                 <div className='argo-form-row'>
-                                    <FormField formApi={formApi} label='Enable LFS support' field='enableLfs' component={CheckboxField}/>
+                                    <FormField formApi={formApi} label='Enable LFS support (Git only)' field='enableLfs' component={CheckboxField}/>
```

Comment:
I think we agreed to show fields based on selected type. So `Enable LFS support` should be hidden unless Git is selected

Reply:
Hai. This PR is a dumping ground for any Helm polish.

Reply:
How about labelling it with "(Git only)" for the moment?

Reply:
sure.